std/math.d

```diff
@@ -307,6 +311,9 @@ template floatTraits(T)
                 enum EXPPOS_SHORT = 0;
                 enum SIGNPOS_BYTE = 0;
             }
+
+            import std.typecons : Tuple;
+            alias Integral = Tuple!(ulong, ushort);
```

Comment:
This should be the same as ``mant_dig == 64``.  All bit manipulation is identical to ``ieeeExtended`` for this type.